### DISCUSSION TITLE
Openpose edit button is not responding.

### DISCUSSION THREAD

**Discussion by AlpDindar** [Mikubill/sd-webui-controlnet#3010] [General]
edit button that worked yesterday does not work today. I uninstalled and reinstalled the plugin but it doesn't work.

I would be grateful if you can help me.

**Comment by Zbramus** [comment]
Hi there, had the same problem,
I just renamed the folder "sd-webui-controlnet" inside "stable-diffusion-webui\extensions" and restarted A1111 and now it works.
Don't really know why though...

**Reply by AlpDindar** [reply]
Thank you for your interest, but it didn't work out for me.

**Reply by Zbramus** [reply]
Weird... You can check if have the "openpose editor" extension installed, If you have, delete it.
My controlnet redownloaded some .pth files and now it works.

**Reply by AlpDindar** [reply]
Thank you very much. I will try again, in which folder do you have the models?

**Reply by Zbramus** [reply]
Np ;)
If you mean the folders where you drop the canny or lineart etc models it's "extensions\sd-webui-controlnet\models"
If you're talking about the models controlnet redownloaded for me it was automatic after I pressed the "run preprocessor" (explosion like) button. But I believe it was in the "extensions\sd-webui-controlnet\annotator\download"

**Comment by norepyh** [comment]
I am having the same issue. I am using A1111 v1.9.4, Python 3.10.11. I have updated sd-webui-controlnet two weeks ago or so and since then the embedded openpose "edit" button isn't working.

I have tried to remove the sd-webui-openpose-editor folder, then restart A1111, but controlnet doesn't seem to reload the plug-in when I hit the edit button.
I have then reinstalled sd-webui-openpose-editor, once again with the same negative results.

It seems controlnet isn't connecting properly to sd-webui-openpose-editor since of its last update. Any workaround while a fix is provided?

**Reply by Zbramus** [reply]
I have completely uninstalled sd-webui-openpose-editor and it's working for me...
You can also try to remove this controlnet and install one from another fork which predates the last update

**Reply by AlpDindar** [reply]
I've tried everything but it doesn't work. Do you know how to install the version prior to the latest update?

**Reply by CeroxeAnivie** [reply]
我解决了这个问题！替换为最新的1.10.0-RC没有这个毛病！！！
![dc5cd37742d905c1c11c57ef169af53](https://github.com/user-attachments/assets/9ffded68-3c4f-4cb2-88d0-fe472ded7120)
